.github/workflows/pr_or_main_test_run.yml

```diff
@@ -32,6 +32,5 @@ jobs:
         run: |
           npx playwright install
 
-      - name: Run Playwright tests
-        continue-on-error: true
+      - name: Run tests
```

Comment:
In a PR context we don't want to continue-on-error, we want failing tests to show as failing.